### PR TITLE
Update model-builders-inline.md

### PR DESCRIPTION
**Pull request by domsinclair**

### CODE REVIEW

File: docs/guide/model-builders-inline.md [hunk 1]
```diff
@@ -4,45 +4,49 @@ title: ModelBuilders - Inline approach
 
 # ModelBuilders - Inline approach
 
-The first way you can leverage `ModelBuilders` in your code is inline. This approach is very useful if you apply just a couple of attributes to your model. It's a quick and dirty way without going through the ceremony of writing a buddy type for your class. 
+This approach is very useful when applying  just a couple of attributes to a business object. It is both quick and it avoids the requirement to create an additional  (buddy)class.
 
 ## Class Level
 
-We will focus now on class level attributes using the inline approach.
+The following code samples illustrate the use of inline ModelBuilders to replace Class Level attributes.
```
Comment (Paramethod) [inline docs/guide/model-builders-inline.md:11]: Capitalization of:
replace _Class Level_ attributes.

Comment (biohazard999) [reply docs/guide/model-builders-inline.md:11]: 👍

**Comment by biohazard999** [comment]
Besides the `class level` casing, awesome! :shipit:

File: docs/guide/model-builders-inline.md (hunk 1 above)
Comment (domsinclair) [reply docs/guide/model-builders-inline.md:11]: yep

**Pull request merged**